tools/wpt/run.py

```diff
@@ -544,7 +544,13 @@ def setup_kwargs(self, kwargs):
                            "Provide '--mojojs-path' explicitly instead.")
             logger.warning("MojoJS is disabled for this run.")
 
-        kwargs["enable_webtransport_h3"] = True
+        # Never pause after test, since headless shell is not interactive.
```

Comment:
Do not understand why we need special treatment for headless shell. We can use chrome://inspect to inspect UI for headless shell.

Reply:
Can you elaborate on how I can use `chrome://inspect` to [unblock headless shell](https://source.chromium.org/chromium/chromium/src/+/main:third_party/wpt_tools/wpt/tools/wptrunner/wptrunner/executors/executorwebdriver.py;l=81-101;drc=798525f7c43c03197d16cef19df35b205c36ac71;bpv=0;bpt=0)?

Normally, [`--headless` forces `--no-pause-after-test`](https://source.chromium.org/chromium/chromium/src/+/main:third_party/wpt_tools/wpt/tools/wptrunner/wptrunner/wptrunner.py;l=166-167;drc=5852b621a8234c7982b4a7a775e5b7b9a5e17568;bpv=0;bpt=0) because there's no graphical window to close, which would stop the browser.

In a terminal environment, the only way to stop a browser under test is to send a signal, which is not friendly to users. wptrunner itself doesn't have such an interactive capability.